sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.Metrics;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal static class Statsbeat
+    {
+        private static readonly Meter s_myMeter = new("AttachStatsBeatMeter", "1.0");
+
+        private const string StatsBeat_ConnectionString = "<StatsBeat_ConnectionString>";
+
+        internal const int AttachStatsBeatInterval = 86400000;
+
+        static Statsbeat()
+        {
+            s_myMeter.CreateObservableGauge("AttachStatsBeat", () => GetAttachStatsBeat());
+
+            // Configure for attach statsbeat which has collection
+            // schedule of 24 hrs == 86400000 milliseconds.
+            var exporterOptions = new AzureMonitorExporterOptions();
+            exporterOptions.DisableOfflineStorage = true;
+            exporterOptions.ConnectionString = StatsBeat_ConnectionString;
+
+            Sdk.CreateMeterProviderBuilder()
```

Comment:
save the provider to a variable, and possibly call dispose at shutdown

Reply:
Out of curiosity, what would happen if the meter provider is not disposed at shutdown?

Reply:
nothing.
if we run for 23 hours and shut down, dispose will ensure we emit one metric. If the apps never run for 24hours, then dispose ensures metric is emitted atleast once in process lifetime, when it is about to die.